cocos/2d/CCLabel.cpp

```diff
@@ -144,6 +144,11 @@ class LabelLetter : public Sprite
 
         _textureAtlas->updateQuad(&_quad, _atlasIndex);
     }
+
+    //LabelLetter is not draw self directly.
```

Comment:
Do you mean LabelLetter doesn't need to draw themselves? The comments have grammar error here.